cogs/custom.py

```diff
@@ -38,14 +42,73 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
             raise BadArgument("Please provide at least 2 options.")
         if len(options) > 20:
             raise BadArgument("I can only handle 20 options!")
-
         codepoint_start = 127462  # represents "regional_indicator_a" unicode value
         options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=codepoint_start)}
         embed = Embed(title=title, description="\n".join(options.values()))
         message = await ctx.send(embed=embed)
         for reaction in options:
             await message.add_reaction(reaction)
 
+    @command()
+    async def members(self, ctx: Context) -> None:
+        """Returns the number of members in a server."""
+        await ctx.send(embed=Embed(title="Member count", description=ctx.guild.member_count, color=Color.dark_purple()))
```

Comment:
Nice work, but you might consider using a nicer embed with something like statuses of users (how many users are idle, online, DND, offline, maybe even the bot count)